module/motion/QuinticWalk/src/WalkEngine.cpp

```diff
@@ -679,27 +679,21 @@ namespace module::motion {
         trunk_axis_acc_at_last.setZero();
     }
 
-    void QuinticWalkEngine::computeCartesianPosition(Eigen::Vector3f& trunkPos,
-                                                     Eigen::Vector3f& trunkAxis,
-                                                     Eigen::Vector3f& footPos,
-                                                     Eigen::Vector3f& footAxis,
-                                                     bool isLeftsupportFoot) {
-        // Compute trajectories time
-        float time = getTrajsTime();
+    using PositionSupportTuple = std::tuple<Eigen::Vector3f, Eigen::Vector3f, Eigen::Vector3f, Eigen::Vector3f, bool>;
 
-        computeCartesianPositionAtTime(trunkPos, trunkAxis, footPos, footAxis, isLeftsupportFoot, time);
+    PositionSupportTuple QuinticWalkEngine::computeCartesianPosition() const {
```

Comment:
```suggestion
    QuinticWalkEngine::PositionSupportTuple QuinticWalkEngine::computeCartesianPosition() const {
```

Reply:
Are these needed to make it compile?

Reply:
If we remove the duplicate using statement it shouldn't compile.

Reply:
This suggestion is in relation to the previous one.